.github/workflows/ci.yml

```diff
@@ -340,6 +340,7 @@ jobs:
           - default
           - hdp3
           - cdh5
+          - hive3
```

Comment:
i expect CI to fail, eg since we don't have `hive3.1-hive-kerberized` environment yet.

let's see if this is the only problem.

Reply:
@findepi 
I see there is in fact an error related to this.
`
2021-09-02T09:15:11.0037841Z com.github.dockerjava.api.exception.InternalServerErrorException: Status 500: {"message":"Head \"https://ghcr.io/v2/trinodb/testing/hive3.1-hive-kerberized/manifests/41\": denied"}
`
What is more when I try to test locally I get some strange exceptions as well while trying to access hdfs.  Not sure how to proceed with this